<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>카드 정보</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>현재 잔액:</string>
     </property>
    </widget>
    <widget class="QLabel" name="totalLabel">
     <property name="geometry">
      <rect>
       <x>86</x>
       <y>40</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="resetButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>초기화</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>충전</string>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>충전 금액:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="chargeButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>충전</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="chargeEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>결제</string>
    </property>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>결제 금액:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="paymentButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>결제</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="paymentEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
